Building SVT-HEVC does not work

```
08:09:40   Running git clone for SVT-HEVC...
08:09:45 ┌ SVT-HEVC git ............................. [Recently updated]
08:09:45 ├ Running uninstall...
08:09:59 ├ Running cmake...
08:10:11 ├ Running build...
Likely error:
C:/AB-Suite/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/9.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: use the --help option for usage information
collect2.exe: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
[1/1] Linking C executable ..\Bin\Release\SvtHevcEncApp.exe
FAILED: ../Bin/Release/SvtHevcEncApp.exe
cmd.exe /C "cd . && C:\AB-Suite\msys64\mingw64\bin\gcc.exe -mthreads -mtune=generic -O2 -pipe -Wall -Wextra -Wformat -Wformat-security -fstack-protector-strong -fvisibility=hidden -fPIE -fPIC -D_FORTIFY_SOURCE=2 -fno-asynchronous-unwind-tables -O3 -DNDEBUG -O2 -O3  -pipe -static-libgcc -static-libstdc++ -z noexecstack -z relro -z now -pie Source/App/CMakeFiles/SvtHevcEncApp.dir/EbAppConfig.c.o Source/App/CMakeFiles/SvtHevcEncApp.dir/EbAppContext.c.o Source/App/CMakeFiles/SvtHevcEncApp.dir/EbAppFifo.c.o Source/App/CMakeFiles/SvtHevcEncApp.dir/EbAppMain.c.o Source/App/CMakeFiles/SvtHevcEncApp.dir/EbAppProcessCmd.c.o Source/App/CMakeFiles/SvtHevcEncApp.dir/EbTime.c.o Source/App/CMakeFiles/SvtHevcEncApp.dir/EbAppInputy4m.c.o  -o ..\Bin\Release\SvtHevcEncApp.exe -Wl,--major-image-version,0,--minor-image-version,0  ../Bin/Release/libSvtHevcEnc.dll.a -lpthread -lrt -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -luuid -lcomdlg32 -ladvapi32 && cd ."
C:/AB-Suite/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/9.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: unrecognized option '-z'
C:/AB-Suite/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/9.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: use the --help option for usage information
collect2.exe: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
build failed. Check C:/AB-Suite/build/SVT-HEVC-git/build-64bit/ab-suite.build.log
This is required for other packages, so this script will exit.
08:11:29   Creating diagnostics file...

All relevant logs have been anonymously uploaded to https://0x0.st/zeWT.zip
Copy and paste [logs.zip](https://0x0.st/zeWT.zip) in the GitHub issue.
Make sure the suite is up-to-date before reporting an issue. It might've been fixed already.
Try running the build again at a later time.
```

--- a/build/media-suite_helper.sh
+++ b/build/media-suite_helper.sh
@@ -1092,10 +1092,9 @@ do_cmake() {
         *)
             if [[ -d "./$1" ]]; then
                 [[ -n "$skip_build_dir" ]] && root="./$1" || root="../$1"
-            elif [[ -z "$bindir" ]]; then
-                bindir="$1"
+                shift
             fi
-            shift && break ;;
+            break ;;
         esac
     done
 

--- a/build/media-suite_helper.sh
+++ b/build/media-suite_helper.sh
@@ -1080,7 +1080,7 @@ do_custom_patches() {
 do_cmake() {
     local bindir=""
     local root=".."
-    local cmakebuilddir=""
+    local cmake_build_dir=""
     while (( "$#" )); do
         case "$1" in
         global|audio|video)

